### ISSUE TITLE
Demo app is not updading the new repos pushed by contributors, is a gridsome thing

### ISSUE THREAD

**Issue by jurra** [CombatCovid/SPA-website#44]
As I have previously mentioned, Nuxt in the long run was the technolgy to use. Gridsome doesnt react to APIs changes like Nuxt does. I just tested today uploading a new repo, in order for the app to display this, the server has to be restarted......Not a good solution....

....we shall overcome @narration_sd, 

I think is time to start moving to Nuxt.js, is something I can take care off :)

Response from @travis in gridsome community:

travis03/27/2020
@jurra Do you want realtime github API updates? As in the API is called when a user visits the gridsome site?

jurra03/27/2020
yes
that would be ideally

travis03/27/2020
In that case, Gridsome may not be your best option... You wouldn't be able to use the Gridsome store, as it is more for a collection of data that updates infrequently, like a blog site for example.
In your case it would just be acting as a usual Vue app, loading data into Vuex as the user visits the site.
Does that make sense?
as it is more optimised for a collection of data that updates infrequently, like a blog site for example.
I.e. you load that data at build time, and it doesn't change until you next rebuild the site.

jurra03/27/2020
Yep that makes sense
Would Nuxt work for this?

travis03/27/2020
Yes, Nuxt would probably be a better fit - especially with the new fetch api etc.

**Comment by narration-sd** [comment]
Ok, Jose, as in the note I left you on Discord which you'll see in your morning, that you didn't 'see' new repo data has absolutely nothing to do with Gridsome -- or Nuxt either, which would act _precisely_ the same. 

The reason:  all the date the app uses (it isn't in the app, it's on Algolia...) is entirely FAKE at this moment of our alpha development. I created it manually. Next steps indeed will be to build the data from actual Design Repo content, as it is on github api. It's there on my list, now that we have a shiny enough demo client to show for the purposes. And once that Builder works, we will then make it automatically updating. Yes. That auto-build won't have anything to do with any systems like Gridsome or Nuxt.

Automatic updating is an _entirely_ separate operation from anything Gridsome or indeed that N word would do. Probably I need to make a diagram, to show absolute separation.

As far as travis, I'm sure he's bright.  But I think from the way he talks to you, it's the old 'if you have a hammer, everything looks like a nail for it' (design) pattern.   

In a word, his advice on these matters hasn't strategic thinking, and when he talks this way, it's clear he doesn't grasp our very effective architecture, which is  no sin  because it hasn't been communicated to him.  But also, yes, it's not in the tradition these guys are focused on. But as you see, immediately it works...fast, and will live well with data growth, having no server demand when that happens.

Let me know, any thoughts, but will close this for now, imagine you'll agree.

Thanks indeed for the heads-up -- we need more of that explanation as others get interested. But priority goes to things that set off misunderstanding, like this. That's why I did the latest client, so that what people see, doesn't raise questions, does attract enthusiasm. I know you know, we discuss so well all of it, so what's here is just to give a clear answer on the question and its factors to others. Best...
